x/evm/keeper/keeper_test.go

```diff
@@ -392,6 +393,10 @@ func TestBaseKeeper(t *testing.T) {
 		assert.NotPanics(t, func() {
 			_ = ck.GetParams(ctx)
 		})
+		assert.Equal(t, uint64(1), ck.GetRequiredConfirmationHeight(ctx))
+		assert.Equal(t, int64(50), ck.GetRevoteLockingPeriod(ctx))
+		assert.Equal(t, utils.Threshold(utils.Threshold{Numerator: 51, Denominator: 100}), ck.GetVotingThreshold(ctx))
+		assert.Equal(t, int64(1), ck.GetMinVoterCount(ctx))
```

Comment:
these asserts have nothing to do with the thing that should be tested here. The only thing that makes sense is to check that the params contain the expected chain name